config/auth.js

```diff
@@ -0,0 +1,32 @@
+/*eslint-env node*/
```

Comment:
it is not good to put a code in /config. preferably /config should contain only configuration data needed by your application, possibly with permutations by enviroment
all such logic belongs either to /src/model/*, if you setup some model entitites (data could  be read from some json in /config though). or to /src/helpers/, /src/lib/, if you just avoiding repetitive code.

Reply:
moved to model folder